plugins/inputs/postgresql_extensible/postgresql_extensible.go

```diff
@@ -59,6 +60,11 @@ var sampleConfig = `
   ## default is forever (0s)
   max_lifetime = "0s"
 
+  ## Whether to use prepared statements when connecting to the database.
+  ## This should be set to false when connecting through a PgBouncer instance
+  ## with pool_mode set to transaction.
+  prepared_statements = true
```

Comment:
Please comment the option as this is the default
```suggestion
  # prepared_statements = true
```